### PR TITLE
chore(ui): migrate AppIntro to TypeScript

### PR DESCRIPTION
**Pull request by gjaskiewicz**
# Summary

Migrate AppIntro to TypeScript

# Changes Made

- migrate AppIntro to TypeScript

# Testing Instructions

1. `npm i`
2. `npm run storybook`

# Checklist

- [ ] I have performed a self-review of my code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have made corresponding changes to the documentation (if applicable).
- [ ] My changes generate no new warnings or errors.

### CODE REVIEW

**Review by barsukov — APPROVED** [review]
LGTM

File: packages/ui-components/src/components/AppIntro/AppIntro.component.tsx [hunk 1]
```diff
@@ -16,16 +15,16 @@ const introStyles = `
...
   )
 }
 
-AppIntro.propTypes = {
+export interface AppIntroProps extends React.HTMLProps<HTMLDivElement> {
   /** Add custom class name */
-  className: PropTypes.string,
-  children: PropTypes.any,
+  className?: string
+  children?: React.ReactNode
```
Comment (guoda-puidokaite) [inline packages/ui-components/src/components/AppIntro/AppIntro.component.tsx:29]: Update to correct type in Storybook, instead of `any`.

**Review by guoda-puidokaite — CHANGES_REQUESTED** [review]
Please make small type change in Storybook and we can merge asap 💪

**Review by guoda-puidokaite — APPROVED** [review]
🚀

**Review by barsukov — APPROVED** [review]

**Pull request merged**